arrow/arrow.py

```diff
@@ -122,6 +137,7 @@ def now(cls, tzinfo=None):
             dt.second,
             dt.microsecond,
             dt.tzinfo,
+            fold=getattr(dt, "fold", 0),
```

Comment:
We need to change this to conform to the spec mentioned here: https://www.python.org/dev/peps/pep-0495/#what-time-is-it.

Reply:
What about it needs to change?

Reply:
Don't we need to set fold=1 (rather than 0) when now is called without arguments?

Reply:
Well, in this case you are passing through what `datetime.now` does, and setting `fold=0` only if the `fold ` attribute doesn't exist at all.

What that is saying is basically that if `datetime.now(tz.tzlocal())` would set `fold=1`, `datetime.now()` will set `fold=1`. This is useful because you can do something like this: `datetime.now().astimezone()`, and it will get you the right fixed-offset zone even in an ambiguous time.

In Python 3.6+, `datetime.now()` will do that for you. In Python 2, the semantics of naïve datetimes are different anyway and I'm not sure how useful it would be to backport this particular feature. Might still be useful for Python 3.5, though.

If you want to implement this behavior, I think it can be done in terms of `dateutil.tz.tzlocal()` on 2.7 and 3.5, but that might be overkill.

Reply:
Ah thanks for clarifying that, I see :). @systemcatch we should probably add those two examples Paul mentionned in the test cases. We may have to adjust `astimezone()` as well since it seems to currently require a tz argument. We should probably set a default to support this.